tests/ApiCallableTest.php

```diff
@@ -175,9 +175,17 @@ public function testRetryTimeoutExceeds()
             $backoffSettings);
         $callSettings = new CallSettings(['retrySettings' => $retrySettings]);
 
+        // Use time function that simulates 1100ms elapsing with each call to the stub
+        $incrementMillis = 1100;
+        $timeFuncMillis = function() use ($stub, $incrementMillis) {
```

Comment:
Just had a look, I don't really like the look of it... It accepts the code for the new function as a string (why!?), so it can't bind to the stub, it would (I think) need to access the stub from a global variable.